popupwindow.js

```diff
@@ -0,0 +1,111 @@
+const closeButton = document.getElementById('closeButton');
+const body1 = document.querySelector('body');
+
+closeButton.addEventListener('click', (e) => {
+  e.stopPropagation();
+  document.querySelector('#inPop').style.display = 'none';
+  document.querySelector('#popup').style.display = 'none';
+  document.getElementById('popup-content').style.display = 'none';
+  body1.style.overflow = 'scroll';
+});
```

Comment:
- [ ] :thinking: Kindly make sure the event that closes the popup is working properly :recycle: